libcrux-ml-kem/cg/libcrux_core.h

```diff
@@ -25,10 +25,6 @@ typedef struct core_ops_range_Range__size_t_s {
   size_t end;
 } core_ops_range_Range__size_t;
 
-static inline uint8_t Eurydice_bitand_pv_u8(uint8_t *x, uint8_t y);
```

Comment:
I guess these were some kind of Eurydice helpers that are no longer generated?

Reply:
Yes, they are built-ins, generated from eurydice. But they were not caught by the phase that removed unused code. Now they are and hence are removed. See https://github.com/AeneasVerif/eurydice/issues/45